test/functional/api/server_notifications_spec.lua

```diff
@@ -76,6 +77,10 @@ describe('notify', function()
   end)
 
   it('cancels stale events on channel close', function()
+      if isCI() then
+        pending('Hangs builds on sourcehut CI. Skip until it is fixed.')
```

Comment:
It hangs on more than just sourcehut.  How about just

```suggestion
        pending('Sporadic hangs on CI (c.f., #14083). Skip until it is fixed.')
```